webpack.mix.js

```diff
@@ -15,3 +15,21 @@ mix.ts('resources/js/app.ts', 'public/js')
     .vue({ version: 2 })
     .sass('resources/sass/app.scss', 'public/css');
 
+mix.override(webpackConfig => {
+    // BUG: vue-loader doesn't handle file-loader's default esModule:true setting properly causing
+    // <img src="[object module]" /> to be output from vue templates.
```

Comment:
I am not sure what this is for? You aren't importing the image object itself to be used in the source, you are using the url that is generated. Seems to me that file copying from resources to public happens by default: https://laravel.com/docs/8.x/mix#url-processing

Reply:
Wait, no, I'm a bit of a dunce! Now I see why you were using the source image and the webpack loader. -> for asset versioning. Resolving!